src/main/webapp/welcomeStyle.css

```diff
@@ -11,11 +11,12 @@ body {
   margin: 8vh 10vw 5vh 10vw;
   height: 90%;
   cursor: default;
+  font-size: 3vh;
```

Comment:
nit: vh is not usually used for fonts. This will make your fonts smaller if the window shorter and larger if it's taller. It's recommended to use px, em, or rem because the sizes of these only change when the user zooms in or out and is therefore more accessible.

Reply:
As we discussed, I will leave it as is for now and fix this in a later PR.